exasol/analytics/audit/audit.py

```diff
@@ -23,12 +26,28 @@
 )
 
 
+def base_column_values(
```

Comment:
It seems this function does two distinct things - changing the keys and adding another entry.

Reply:
Yes, as described in the docstring. 
Would you prefer a different approach?

Reply:
Personally, I don't take these things religiously, but the convention is that a function should do only one thing.